### DISCUSSION TITLE
import point cloud

### DISCUSSION THREAD

**Discussion by Lolboer9999999** [assimp/assimp#5029] [Q&A]
Hello
When i try to load a 3d model that only contains vertices, mNumMeshes returns 0. I've tried with .obj files and .fbx files
obj file looks like this (but with more points)
# Blender 3.4.0
# www.blender.org
o Main_Layer
v 0.859694 0.255555 -0.750696
v 0.999988 0.395849 -0.750696
v 0.859694 0.255555 -1.590696
v 0.859694 0.255555 1.099918

Loading a regular object works fine. Does anyone know how to do this?
(im using the library in a C program)

**Comment by kimkulling** [comment]
You need to use STL as the import schema. This is the only one who supports point clouds.
